en/flight_controller/nxp_mr_vmu_rt1176.md

```diff
@@ -0,0 +1,267 @@
+# NXP MR-VMU-RT1176 Flight Controller
+
+<Badge type="tip" text="PX4 v1.15" />
+
+:::warning
+PX4 does not manufacture this (or any) autopilot.
+Contact the [manufacturer](https://www.nxp.com) for hardware support (https://community.nxp.com/) or compliance issues.
+:::
+
+:::tip
+This autopilot reference design is [supported](../flight_controller/autopilot_manufacturer_supported.md) by NXP.
+:::
+
+The _MR-VMU-RT1176_ reference design is based on the Pixhawk<sup>&reg;</sup> FMUv6X-RT open standard, the latest update to the successful family of Pixhawk<sup>&reg;</sup> flight controllers.
+
+The board includes the same FMU module found on Pixhawk 6X-RT paired with an NXP-based carrier board.
+The carrier board offers 100Base-T1 (two-wire) automotive ethernet, an NFC antenna (Connected to SE051), and a third CAN bus.
+It also removes the IO processor to enable 12 PWM ports, with 8 providing Dshot capability.
+
+![MR-VMU-RT1176 Upright Image](../../assets/flight_controller/nxp_mr-vmu-rt1176/mr-vmu-rt1176_upleft.jpg)
+
+This is NXP's open source _reference design_ for using FMUv6X-RT, and was designed and made in collaboration with industry partners<sup>&reg;</sup>, NXP's mobile robotics team and the PX4 team.
+As a reference/evaluation design, it is intended to be replicated, modified or integrated in production volume by others.
+Several 3rd party manufacturers (such as Holybro.com) provide this or derivative commercial products.
+Note that NXP is a semiconductor vendor, and since it is a reference design, this particular board is not produced in high volume.
+
+This board takes advantage of multiple Pixhawk​​® open standards, such as the FMUv6X-RT Standard, [Autopilot Bus Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-010%20Pixhawk%20Autopilot%20Bus%20Standard.pdf), and [Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf).
+Equipped with a high performance NXP i.mx RT1176 dual core Processor, modular design, triple redundancy, temperature-controlled IMU board, isolated sensor domains, delivering incredible performance, reliability, and flexibility.
+
```

Comment:
:::tip
This autopilot is [supported](../flight_controller/autopilot_pixhawk_standard.md) by the PX4 maintenance and test teams.
:::